app/models/Conference.php

```diff
@@ -26,7 +26,14 @@ public function author()
 
     public function cfpIsOpen()
```

Comment:
@adamwathan  Is there any reason to keep `cfpIsOpen` around if we're just passing it through?